src/css/components/_progress.scss

```diff
@@ -97,3 +97,19 @@ specific time ranges that have been buffered */
 .video-js.vjs-no-flex .vjs-progress-control {
   width: auto;
 }
+
+.video-js .vjs-progress-control .vjs-mouse-display {
+  visibility: hidden;
+  position: absolute;
+  right: 0;
+  left: 0;
+  top: calc(50% - 0.15em);
```

Comment:
`calc` is terrible support, so, I'll probably need to swap it out to something else.